core/src/main/java/org/elasticsearch/index/query/DisMaxQueryBuilder.java

```diff
@@ -54,6 +61,24 @@ public DisMaxQueryBuilder add(QueryBuilder queryBuilder) {
     }
 
     /**
+     * Add a list of sub-queries to this disjunction.
+     */
+    public DisMaxQueryBuilder add(List<QueryBuilder> queryBuilders) {
```

Comment:
ok no big deal we can keep it if it makes sense for consistency. Or if we added those list methods to bool query builder maybe we should go back and remove them? less code is always better, especially if it can be avoided by looping. Allowing a List makes me wonder why we don't support Collection and varargs too, which would mean too many add methods :)